docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,286 @@
 ---
 title: Teleport Zero Trust Access
 description: Provides guides for the Teleport Zero Trust Access product.
+template: "landing-page"
 ---
 
-<DocCardList />
+import LandingHero, { LandingHeroProps } from '@site/src/components/Pages/Landing/LandingHero';
+import Resources, { ResourcesProps } from "@site/src/components/Pages/Homepage/Resources";
+import UseCasesList, { UseCasesListProps } from "@site/src/components/Pages/Landing/UseCasesList";
+
+import zeroTrustAccessImg from '@version/docs/img/zero-trust-access/zta-hero.png';
+import applicationSvg from "@site/src/components/Icon/teleport-svg/application.svg";
+import linuxServersSvg from "@site/src/components/Icon/teleport-svg/linux-servers.svg";
+import databaseSvg from "@site/src/components/Icon/teleport-svg/database-access.svg";
+import kubernetesClustersSvg from "@site/src/components/Icon/teleport-svg/kubernetes-clusters.svg";
+import windowsDesktopsSvg from "@site/src/components/Icon/teleport-svg/windows-desktops.svg";
+import autoDiscoverySvg from "@site/src/components/Icon/teleport-svg/auto-discovery.svg";
+import cloudProvidersSvg from "@site/src/components/Icon/teleport-svg/cloud-providers.svg";
+import mcpAndAiSvg from "@site/src/components/Icon/teleport-svg/mcp-and-ai.svg";
+
+<LandingHero
+  title="Zero Trust Access"
+  image={zeroTrustAccessImg}
+>
+  Zero Trust Access provides engineers with just-in-time, least‐privileged
+  access to applications, servers, databases, Kubernetes clusters, and other
+  resources across distributed infrastructures, improving engineering time to
+  market and infrastructure resiliency.
+</LandingHero>
+
+<UseCasesList
+  title="Popular Actions"
+  desktopColumnsCount={2}
+  useCases={[
+    {
+      title: "Sign In with Your Identity Provider",
+      description: "Log in to servers, Kubernetes clusters, databases, web applications, and Windows desktops through their organization's Single Sign-On (SSO) provider.",
+      href: "./sso/",
+      tags: [
+        {
+          name: "Okta",
+          href: "./sso/okta/",
+          icon: "okta",
+        },
+        {
+          name: "GitHub",
+          href: "./sso/github-sso/",
+          icon: "github",
+        },
+        {
+          name: "Entra ID",
+          href: "./sso/entra-id-oidc/",
+          icon: "entraId",
+        },
+        {
+          name: "G Suite",
+          href: "./sso/google-workspace/",
+          icon: "googleCloud",
+        },
+       {
+          name: "more",
+          href: "./sso/",
+          arrow: true,
+        },
+      ]
+    },  
+    {
+      title: "Manage and Track Teleport Audit Events",
+      description: "View audit log generated when users and services interact with your cluster. And export events to store them in a log management platform.",
```

Comment:
_And export events to store them in a log management platform._ doesn't feel like a complete sentence.